tensorflow/compiler/xla/service/cpu/cpu_compiler.h

```diff
@@ -29,6 +29,14 @@ limitations under the License.
 #include "tensorflow/core/platform/macros.h"
 #include "tensorflow/core/platform/stream_executor_no_cuda.h"
 
+#if (defined(__powerpc__) || defined(__ppc__) && (__BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__))
+#define TARGET_CPU "ppc"
+#define TARGET_TRIPLE "ppc64le-ibm-linux-gnu"
+#else
+#define TARGET_CPU ""
+#define TARGET_TRIPLE "x86_64-pc-linux"
+#endif
+
```

Comment:
This should not live in cpu_compiler.h -- we cannot use `TARGET_CPU` and `TARGET_TRIPLE` in the compiler because that won't work for the AOT cross compilation mode.  Using these definitions in tests is fine.

I'd suggest putting it into a `test_target_triple_helper.h` header or something like that and mark the bazel target as `testonly`.

Moreover, I don't see why these need to be macros.  Let's make these normal globals instead, like `kTargetCpuForHost` etc.?

Reply:
Sure, I'll make this change.